src/app/core/models/rundown-event.ts

```diff
@@ -58,6 +59,7 @@ export interface RundownPieceInsertedEvent extends PartEvent {
 export interface RundownCreatedEvent extends RundownEvent {
   type: RundownEventType.RUNDOWN_CREATED
   basicRundown: BasicRundown
+  rundown: Rundown
```

Comment:
Since we now have the Rundown, I don't think we need the BasicRundown anymore, since Rundown has all the information that BasicRundown have.

Reply:
This has been removed :-)